nix/deps/gradle/generate.sh

```diff
@@ -83,9 +83,15 @@ fi
 gen_proj_list
 gen_deps_list
 # FIXME: Temporary fix for missing package.
-echo 'org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin:1.6.10' >> "${DEPS_LIST}"
-echo 'com.android.tools.lint:lint-gradle:30.2.2' >> "${DEPS_LIST}"
-echo 'com.android.tools.build:gradle:7.0.4' >> "${DEPS_LIST}"
+echo 'com.android.tools.build:gradle:1.3.1' >> "${DEPS_LIST}"
+echo 'com.squareup.okio:okio:1.13.0' >> "${DEPS_LIST}"
+echo 'com.squareup.okio:okio:1.15.0' >> "${DEPS_LIST}"
+echo 'com.squareup.okhttp3:okhttp:3.12.1' >> "${DEPS_LIST}"
+echo 'org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin:1.7.22' >> "${DEPS_LIST}"
+echo 'com.android.tools.lint:lint-gradle:30.4.2' >> "${DEPS_LIST}"
+echo 'com.android.tools.build:gradle:7.4.2' >> "${DEPS_LIST}"
+# Required by configuration ':react-native-async-storage_async-storage:detachedConfiguration2'.
+echo 'com.android.tools.build:aapt2:7.4.2-8841542' >> "${DEPS_LIST}"
```

Comment:
Merciful Christ. This is awful. Are we sure we need all of those? We seriously need to deal with this issue:

* https://github.com/status-im/status-mobile/issues/15447

Reply:
I agree. Do we have at lease a mechanism to alert if version is not in sync with required one?
Not to say, that we will still download a lib even if we stop using it, but I guess it's minor.